modules/nback.llvm/llvm.bal

```diff
@@ -7,14 +7,14 @@
 
 import ballerina/io;
 
-public type IntType "i8"|"i1";
+public type IntType "i8"|"i1"|"i64";
```

Comment:
i8 is my mistake. We don't need it. Just replace by i64

Reply:
Fixed